system/Session/Handlers/RedisHandler.php

```diff
@@ -142,17 +142,19 @@ protected function setSavePath(): void
             }
         }
 
-        $password = $query['auth'] ?? null;
-        $database = isset($query['database']) ? (int) $query['database'] : 0;
-        $timeout  = isset($query['timeout']) ? (float) $query['timeout'] : 0.0;
-        $prefix   = $query['prefix'] ?? null;
+        $persistent = isset($query['persistent']) ? (bool) $query['persistent'] : null;
```

Comment:
Maybe we can do something like `filter_var($query['persistent'], FILTER_VALIDATE_BOOLEAN)` instead of just casting to `bool`?